notifications_utils/safe_string.py

```diff
@@ -2,7 +2,7 @@
 import unicodedata
 
 
-def email_safe(string, whitespace='.'):
+def make_string_safe(string, whitespace='.'):
```

Comment:
I think we can drop the default argument for whitespace in here?

Reply:
```suggestion
def make_string_safe(string, whitespace):
```